CHANGELOG.md

```diff
@@ -34,7 +55,6 @@ All notable changes to this project will be documented in this file. The format
 - **Docs:**
   - Revised the SpamAssassin ENV docs to better communicate configuration and their relation to other ENV settings. ([#3756](https://github.com/docker-mailserver/docker-mailserver/pull/3756))
 
-
```

Comment:
Likewise, not something this PR should be modifying.

Reply:
You're right, but for this PR, not a problem now IMO.